integrationTests/sync/common.go

```diff
@@ -0,0 +1,182 @@
+package sync
+
+import (
+	"context"
+	"fmt"
+	"sync/atomic"
+	"time"
+
+	"github.com/ElrondNetwork/elrond-go/dataRetriever"
+	"github.com/ElrondNetwork/elrond-go/integrationTests"
+	"github.com/ElrondNetwork/elrond-go/p2p"
+	"github.com/ElrondNetwork/elrond-go/sharding"
+)
+
+var stepDelay = time.Second
+var delayP2pBootstrap = time.Second * 2
+var stepSync = time.Second * 2
+
+func setupSyncNodesOneShardAndMeta(
+	numNodesPerShard int,
+	numNodesMeta int) ([]*integrationTests.TestProcessorNode, p2p.Messenger, []int) {
```

Comment:
arrange function input and output, put output in new line

Reply:
Done